src/main/java/uk/gov/digital/ho/hocs/aws/MigrationSqsConfiguration.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.digital.ho.hocs.aws;
```

Comment:
Likely isn't required if you use the main queue.

Reply:
Removed